lib/KFocusTrap.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+
+  <div>
+    <!-- Focus trap starting point. If not disabled, focuses this first when tabbing. -->
+    <div
+      v-if="!disabled"
+      class="focus-trap-first"
+      tabindex="0"
+      @focus="handleFirstTrapFocus"
+    ></div>
+
+    <!-- Default slot where the focusable content will be rendered -->
```

Comment:
This should be the below so that the description for the slot is added in the component's documentation. I've added `@slot`.
```vue
    <!-- @slot Default slot where the focusable content will be rendered -->
```